examples/cqlsh-rs.rs

```diff
@@ -1,10 +1,178 @@
 use anyhow::Result;
+use rustyline::completion::{Completer, Pair};
 use rustyline::error::ReadlineError;
-use rustyline::Editor;
+use rustyline::{CompletionType, Config, Context, Editor};
+use rustyline_derive::{Helper, Highlighter, Hinter, Validator};
 use scylla::transport::Compression;
 use scylla::{QueryResult, Session, SessionBuilder};
 use std::env;
 
+#[derive(Helper, Highlighter, Validator, Hinter)]
+struct CqlHelper {}
+
+const CQL_KEYWORDS: &[&str; 123] = &[
```

Comment:
Let's drop the table size and just convert it into a slice, otherwise it will be annoying to update the size every time somebody adds a new keyword. This should work:

```rust
const CQL_KEYWORDS: &[&str] = &[/* ... */];
```

Reply:
thx, done